src/main/java/net/asodev/islandutils/mixins/cosmetics/FontLoaderMixin.java

```diff
@@ -39,7 +40,9 @@ private void init(ResourceLocation file, int i, int j, int[][] chars, CallbackIn
                 case "_fonts/tooltips/hat.png" -> CosmeticState.HAT_COMP = comp;
                 case "_fonts/tooltips/accessory.png" -> CosmeticState.ACCESSORY_COMP = comp;
                 case "_fonts/tooltips/hair.png" -> CosmeticState.HAIR_COMP = comp;
-                case "_fonts/medals.png" -> LevelTimer.medalCharacter = character;
+                case "_fonts/medals.png" -> {
+                    if (MccIslandState.isOnline() && LevelTimer.medalCharacter.isEmpty()) LevelTimer.medalCharacter = character;
```

Comment:
this is certainly a solution, but it is also an edge case
i'm gonna take a look at it and see if there's a bit of a better, more concrete solution that won't make this susceptible to happening again with a different character.

Reply:
I'm going to have a similar implementation and extract the real characters without a dummy resource pack, which should negate the need for any changes on your end. I coded this as a quick fix since I can't release an update for some time. The medal is the only character I'm using that conflicts with IslandUtils.

On second thought, however, I think my pack loaded after because of alphabetical order, so other mods might potentially still cause similar conflicts. I tried to find a way to check the origin resource pack of the glyph but I got nowhere.

Reply:
Apparently I did something that can fix this issue a while ago, and I completely forgot about it:

https://github.com/anastarawneh/MCCINametagMod/blob/e220774376f366bd981f023927827676c104ba78/src/main/java/me/anastarawneh/mccinametagmod/util/Util.java

You can read the actual `font/icon.json` file and parse it manually. The chances that another mod is also using the same `mcc:fonts/icon.json` identifier is incredibly low, so there should be no overlap.

Regardless, I fixed the issue on my end, and I will upload an updated version soon.